### PR TITLE
better formatting for external libraries listing

### PR DESCRIPTION
**Pull request by incanus**
/cc @1ec5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,20 +6,23 @@ A library for embedding interactive, customizable vector maps into native applic
...
+| [Mapbox Qt SDK](platform/qt)            | C++03                              | [![Travis](https://travis-ci.org/mapbox/mapbox-gl-native.svg?branch=master)](https://travis-ci.org/mapbox/mapbox-gl-native/builds) [![Bitrise](https://www.bitrise.io/app/96cfbc97e0245c22.svg?token=GxsqIOGPXhn0F23sSVSsYA&branch=master)](https://www.bitrise.io/app/96cfbc97e0245c22) |
 
 Additional Mapbox GL Native–based libraries are developed outside of this repository:
 
-* [React Native Mapbox GL](https://github.com/mapbox/react-native-mapbox-gl) for React Native applications on iOS and Android
-* Telerik’s [Mapbox plugin](http://plugins.telerik.com/cordova/plugin/mapbox) for Apache Cordova Hybrid applications
-* Telerik’s [Mapbox plugin](http://plugins.telerik.com/nativescript/plugin/mapbox) for NativeScript Hybrid applications
-* Xamarin's [Mapbox component](https://components.xamarin.com/view/mapboxsdk) for Xamarin Hybrid applications
+| Library                                  | Use                                      |     By Mapbox?     |
```
Comment (1ec5) [inline README.md]: “By Mapbox” does emphasize that the unchecked libraries are third-party libraries, but it also implies that RNMBGL is an official Mapbox product, which isn’t the case currently.

**Comment by incanus** [comment]
Suggestion? I was thinking maybe to make the column “Creator” or similar instead, and either list Mapbox or whomever. Does that read better?

**Comment by incanus** [comment]
Removed the author column per chat @1ec5.

File: README.md [hunk 2]
```diff
@@ -15,14 +15,13 @@ This repository hosts the cross-platform Mapbox GL Native library, plus convenie
 | [node-mapbox-gl-native](platform/node/) | Node.js                            | [![Linux](https://travis-ci.org/mapbox/mapbox-gl-native.svg?branch=master)](https://travis-ci.org/mapbox/mapbox-gl-native/builds) [![macOS](https://www.bitrise.io/app/55e3a9bf71202106.svg?token=5qf5ZUcKVN3LDnHhW7rO0w)](https://www.bitrise.io/app/55e3a9bf71202106) |
 | [Mapbox Qt SDK](platform/qt)            | C++03                              | [![Travis](https://travis-ci.org/mapbox/mapbox-gl-native.svg?branch=master)](https://travis-ci.org/mapbox/mapbox-gl-native/builds) [![Bitrise](https://www.bitrise.io/app/96cfbc97e0245c22.svg?token=GxsqIOGPXhn0F23sSVSsYA&branch=master)](https://www.bitrise.io/app/96cfbc97e0245c22) |
 
-Additional Mapbox GL Native–based libraries are developed outside of this repository:
+Additional Mapbox GL Native–based libraries for **hybrid applications** are developed outside of this repository:
```
Comment (incanus) [inline README.md]: 👍

**Pull request merged**